packages/preview-middleware/src/base/flp.ts

```diff
@@ -0,0 +1,231 @@
+import type { ReaderCollection } from '@ui5/fs';
+import { render } from 'ejs';
+import type { Request, Response } from 'express';
+import { readFileSync } from 'fs';
+import { join } from 'path';
+import type { App, FlpConfig } from '../types';
+import { Router as createRouter, static as serveStatic } from 'express';
+import type { Logger } from '@sap-ux/logger';
+import { deleteChange, readChanges, writeChange } from './flex';
+import type { MiddlewareUtils } from '@ui5/server';
+import type { Manifest, UI5FlexLayer } from '@sap-ux/project-access';
+import { json } from 'express';
```

Comment:
We can move this import to line 7

Reply:
https://github.com/SAP/open-ux-tools/pull/1093/commits/0c0b26cacf59f4ead932600574c40e3cfafbe2f2

Reply:
thx